code/__DEFINES/security.dm

```diff
@@ -0,0 +1,40 @@
+#define DETSCAN_CATEGORY_FINGERS "Prints"
+#define DETSCAN_CATEGORY_BLOOD "Blood"
+#define DETSCAN_CATEGORY_FIBER "Fibers"
+#define DETSCAN_CATEGORY_DRINK "Reagents"
+#define DETSCAN_CATEGORY_ACCESS "ID Access"
+
+// custom categories go here
+#define DETSCAN_CATEGORY_ADD "Additional Notes"
+#define DETSCAN_CATEGORY_ILL "Illegal Tech"
+#define DETSCAN_CATEGORY_SYN "Syndicate Tech"
+#define DETSCAN_CATEGORY_HOLY "Holy Data"
+#define DETSCAN_CATEGORY_SET "Active"
```

Comment:
Why SET and not ACTIVE?

Reply:
settings

Reply:
Do you see how this is confusing

Reply:
I also don't even see why this is settings